Modules/Module_Import/NII2BIDS/xASL_bids_PhoenixProtocolAnalyzer.m

```diff
@@ -151,6 +153,10 @@
         bidsPar.ScanType = 'M0';
 	end
     
+	if isfield(sourcePar,'sProtConsistencyInfotBaselineString') && ~isempty(sourcePar.sProtConsistencyInfotBaselineString)
```

Comment:
All nice, just a comment, you know that nobody is able to read your code right?
I mean I can understand a bit what you're doing, but I don't know what each field consists. So if you want you could add a comment that tells the reader what each field does, so others could read this code as well if you would become ill or decide to stop working ;)

Just a comment, you can resolve

Reply:
Good point. Comments added.